torchax/torchax/mesh_util.py

```diff
@@ -0,0 +1,69 @@
+import jax
+from jax.sharding import PartitionSpec, NamedSharding
+import torch
+import torchax
+from torchax import interop
+
+
+def _shard_first_multiple_of(axis_name, shape, multiple_of):
+  sharding = []
+  for size in shape:
```

Comment:
As written, this for loop is a no-op

I think you meant

```
found = False
for size in shape:
  if size % multiple_of == 0 and not found:
    sharding.append(axis_name)
    found = True
  else:
    sharding.append(None)
```

or something like that?

It's unclear to me how the unit test passes

Reply:
good catch!